docs/versionSelector.js

```diff
@@ -0,0 +1,69 @@
+import {
+  LitElement,
+  css,
+  html,
+} from "https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js";
```

Comment:
this is https://lit.dev, a web components framework from Google -- we can vendor it if necessary, but idk if there's a good reason for it.

React, Preact, etc would also work, but I didn't want an extra transpilation step for .jsx. With Lit, I don't need any JS tools on my machine at all.